apps/cli/src/commands/briefs.command.ts

```diff
@@ -179,29 +179,62 @@ Note: Briefs must be created through the Hamster Studio web interface.
 				process.exit(1);
 			}
 
-			// Use the bridge to list briefs
-			const remoteResult = await tryListTagsViaRemote({
```

Comment:
if removing "tryListTagsViaRemote" I think this is the only place where it is used, hence you could maybe remove it, but check twice where it is removed before removing it.